aws-bom/build.gradle

```diff
@@ -0,0 +1,15 @@
+javaPlatform {
+    allowDependencies()
+}
+dependencies {
+    constraints {
```

Comment:
Shouldn't the AWS sdk version etc. be here as well?

Reply:
it is present in the [sdk module](https://github.com/micronaut-projects/micronaut-aws/blob/master/aws-sdk-v2/build.gradle#L6) so I think that's enough and the modules are either using v1 or v2, so question is whether we want to spoil every module with both sdk boms. 

Note: The bom is not defined for v1, so I'll add it there